core/src/datayoga_core/job.py

```diff
@@ -118,11 +118,9 @@ def transform(self, data: List[Dict[str, Any]], deepcopy: boolean = True) -> Job
             result.processed = [Result(Status.SUCCESS, payload=row) for row in transformed_data]
         finally:
             # close the event loop
-            try:
-                loop.close()
-            except NotImplementedError as noe:
+            with suppress(NotImplementedError):
                 # for pyodide. doesn't implement loop.close, ignore
-                pass
+                loop.close()
```

Comment:
see comment above. this will break pyodide I think

Reply:
Yep. I just replaced `try`/`except` clauses to more convenient `suppress`.